### ISSUE TITLE
Week 2: Conditionals and Comprehensions 

### ISSUE THREAD

**Issue by apjama** [app-ren-teec/python-beginners#2]
Hi team, 

This week we're building on the stuff we learnt last week. We're also adding a few more cool things like if conditionals. 

Here are a series of tasks I want you to do: 

1. Read this: https://www.w3schools.com/python/python_conditions.asp 
2. Do this Kata. https://www.codewars.com/kata/557cd6882bfa3c8a9f0000c1/train/python. This one tests stuff you did last week (indices, strings, etc). 
3. Do this Kata: https://www.codewars.com/kata/577ff15ad648a14b780000e7. It tests your knowledge of dictionaries.  
3. Do this Kata: https://www.codewars.com/kata/5b049d57de4c7f6a6c0001d7. This one tests what you've learnt about `If` statements. 
4. Do this Kata: https://www.codewars.com/kata/56170e844da7c6f647000063. This one also tests your knowledge of conditionals. 

Next Sunday at 3pm(London time). Let's do a one hour session together. We can work on some if/data structures stuff together. Go through these katas and answer any question you might still have.

**Comment by apjama** [comment]
Checking in. It's friday. How is everyone getting on? Don't forget, 3pm on Sunday if you want to join us work through some of these problems together.

**Comment by Zhagi** [comment]
Are we supposed to submit something before Sunday? I was under the impression we were working through the tasks until then.

**Comment by Kamal9-tech** [comment]
Yeah some of these tasks deffinately had me going back to week 1, but they
are good practice i still have to attempt last 2 tasks. Looking forward to
going through them on sunday with everyone.

**Comment by mohamedabdisalam** [comment]
@apjama I'll be a bit late to the session at 3 I'm in some lecture. 

What is the link for it btw?

**Comment by apjama** [comment]
Hey team, 

here 

Topic: PYTHON WEEK 2
Time: Jan 24, 2021 03:00 PM London

Join Zoom Meeting
https://us02web.zoom.us/j/81391645403?pwd=S2RuTSsyTk8wdHlQeDcwaU9XM2pXUT09

Meeting ID: 813 9164 5403
Passcode: eA5WtX
One tap mobile
+13017158592,,81391645403#,,,,*854009# US (Washington DC)
+13126266799,,81391645403#,,,,*854009# US (Chicago)

Dial by your location
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 646 558 8656 US (New York)
        +1 669 900 9128 US (San Jose)
        +1 253 215 8782 US (Tacoma)
Meeting ID: 813 9164 5403
Passcode: 854009
Find your local number: https://us02web.zoom.us/u/kdMS5OAs5l

**Comment by Zhagi** [comment]
join again guys

**Comment by malab1** [comment]
i'd appreciate if you left this ticket open a while longer as I haven't found enough time to go through all the information

**Comment by Mustinelli** [comment]
Salaam everyone,

I have attached my solution to the tasks set. However, I could not work out the apparently-modifying-strings task. I would appreciate if anyone that knows how to do it could explain it to me IA.

Any idea as to when the next task will be set? :)

Thanks


________________________________

**Comment by MohamudMussa** [comment]
done with 3 of them, need to practice more function

https://www.codewars.com/kata/557cd6882bfa3c8a9f0000c1/train/python
https://www.codewars.com/kata/5b049d57de4c7f6a6c0001d7/train/python //
didn't get this one

**Comment by malab1** [comment]
Ok i've just done the first one and I think i've made a mistake, since it says failure https://www.codewars.com/kata/557cd6882bfa3c8a9f0000c1/train/python

**Comment by malab1** [comment]
https://www.codewars.com/kata/577ff15ad648a14b780000e7/train/python

**Comment by malab1** [comment]
im quite lost on the last two
